aboutstyle.css

```diff
@@ -0,0 +1,252 @@
+.main-sect {
+  background-image: url(images/hero_container.svg);
+  background-repeat: repeat;
+  background-size: contain;
+}
+
+#title {
+  text-align: center;
+  padding-top: 10%;
+}
+
+.about-text {
+  margin-top: 36px;
+  font-weight: 400;
+  font-size: 14px;
+  line-height: 25px;
+  background-color: #fff;
+  padding: 20px 12px;
+  text-align: center;
+}
```

Comment:
- [ ] Kindly notice the `font-size` and `color` for texts in the provided design. Please make sure you stick to provided design as much as possible.

### Provided design :point_down: 
![Screenshot from 2023-04-27 21-05-28](https://user-images.githubusercontent.com/91142432/234966865-f0b92619-0804-4633-82e5-2c596e749ed8.png)

### Your design :point_down: 
![Screenshot from 2023-04-27 21-08-46](https://user-images.githubusercontent.com/91142432/234967057-6a65f756-c069-49cf-8d3b-df04c29e9941.png)